frontend/src/tests/components/Commons/CommonsOverview.test.js

```diff
@@ -15,12 +11,12 @@ import leaderboardFixtures from "fixtures/leaderboardFixtures";
 import { apiCurrentUserFixtures } from "fixtures/currentUserFixtures";
 import { systemInfoFixtures } from "fixtures/systemInfoFixtures";
 
-jest.mock("react-router-dom", () => ({
-    ...jest.requireActual("react-router-dom"),
-    useParams: () => ({
-        commonsId: 1
-    })
-}));
+// jest.mock("react-router-dom", () => ({
```

Comment:
Commented out code should be deleted unless there's a good reason for it to stick around